### DISCUSSION TITLE
Please helpme. Is it possible auto Type import from generated type-graphql?

### DISCUSSION THREAD

**Discussion by guhyeon** [MichalLytek/typegraphql-prisma#180] [Q&A]
I'm generate with this prisma schema
`// schema.prisma
generator typegraphql {
  provider = "typegraphql-prisma"
}`

and generated results are in 'node_modules/@generated/type-graphql'
![image](https://user-images.githubusercontent.com/9765614/132649890-ac0949b2-a894-43b3-9fe1-1b772df3c5ec.png)

but when I use type, there are no auto complete and auto imports.

For example,
![image](https://user-images.githubusercontent.com/9765614/132649217-37a1be99-6f41-4989-b5f6-e2a1bbef8efa.png)

I'm using webstorm, if another ide support this, then I am be willing to change IDE.

Please help me.

**Comment by MichalLytek** [comment]
Maybe webstorm has some issues with missing `package.json` for a folder in `node_modules`. Hard to say, never used that, in VSCode it works ok.

**Reply by guhyeon** [reply]
Oh Thanks, It's well with vscode.

**Comment by guhyeon** [comment]
I found node_modules/* are excluded my project.

make @generated directory include projects then it works.

![image](https://user-images.githubusercontent.com/9765614/132678771-80e6a718-a74a-4baf-9c97-7f370390c627.png)

**Comment by JClackett** [comment]
I also cant get auto import to work from the @generated code in node_modules, I am using vscode too. @MichalLytek did you have to do anything to get that to work? 

One thing to note is that I have my api as part of a monorepo and the generated code is being put in the top level node_modules, not sure why this would change anything though
